packages/patternfly-4/react-table/src/components/Table/Body.tsx

```diff
@@ -139,6 +144,7 @@ export const TableBody = ({
         rows={rows as IRow[]}
         onRow={onRow}
         className={className}
+        // eslint-disable-next-line react/no-children-prop
```

Comment:
Can we move children to actually be children?

Reply:
Done